app/models/restaurant.rb

```diff
@@ -16,7 +16,8 @@
 #
 
 class Restaurant < ApplicationRecord
-  enumify :cuisine, [:American, :Asian, :Bakery, :Fast_Food, :Steak, :Sushi, :Vegetarian]
+  E_CUISINES = [:American, :Asian, :Bakery, :Fast_Food, :Steak, :Sushi, :Vegetarian].freeze
```

Comment:
why do you need the freeze here? ([hint](https://github.com/omer-katzir/WeEat/blob/9778634eee5f9c2938a35dba41951132c2506683/.rubocop.yml#L16))

Reply:
not sure what you mean, the FrozenStringLiteralComment = disabled, was also in the default rubocop.yml you showed me,
also tried to enable it and place the magic comment in all the relevant files plus removing the freeze call, still got offense on it